app/views/layout.erb

```diff
@@ -14,15 +14,21 @@
   <%# finally, make the page-top bar somewhat respectably toned down too %>
   <meta name="apple-mobile-web-app-status-bar-style" content="black">
 
-  <script src="jquery.js"></script>
-  <script src="jquery.cookie.js"></script>
-  <script src="pages.js"></script>
+  <!-- <script src="jquery.js"></script> -->
+  <script src="//code.jquery.com/jquery-1.11.2.js"></script>
+  <!-- <script src="jquery.cookie.js"></script> -->
+  <!-- <script src="pages.js"></script> -->
```

Comment:
This file was pretty important because it kept the state of the checkbox consistent between page refreshes. I get that it was broken because the cookie plugin was broken. Instead of setting cookies, you can try using `sessionStorage` to keep the checked state of the checkbox consistent between refreshes.